packages/core/src/Modules/Cashier/Containers/on-ramp.jsx

```diff
@@ -5,8 +5,20 @@ import { Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 import OnRampProviderCard from '../Components/on-ramp-provider-card.jsx';
 import OnRampProviderPopup from '../Components/on-ramp-provider-popup.jsx';
+import SideNote from '../Components/side-note.jsx';
 import 'Sass/app/modules/on-ramp.scss';
 
+const OnRampSideNote = () => {
+    const notes = [
+        <Localize
+            i18n_default_text='Fiat onramp is a cashier service that allows you to convert fiat currencies to crypto to top up your Deriv crypto accounts. Listed here are third-party crypto exchanges. You’ll need to create an account with them to use their services.'
+            key={0}
+        />,
+    ];
+
+    return <SideNote notes={notes} has_bullets={false} title={<Localize i18n_default_text='What is Fiat onramp?' />} />;
```

Comment:
Could we omit `has_bullets` if it's `false`?

Reply:
most of the sidenotes has bullets, so i made has_bullets as default, or do we want it the other way round and omit has_bullets from onramp?

Reply:
Ideally we'd follow the general approach (opt-in), so if it isn't defined, then it shouldn't be there.

Reply:
yups, has omitted 🙏